config.yml

```diff
@@ -54,7 +54,7 @@ defaults:
 
   # Image used to  create the taskcluster proxy container.
   taskclusterProxyImage: 'taskcluster/taskcluster-proxy:latest'
-  taskclusterLogImage: 'taskcluster/livelog:v3'
+  taskclusterLogImage: 'taskcluster/livelog:v4'
```

Comment:
Note, we should probably align on the version number in the software, and the docker tag - e.g. this should probably be v1.0.1 rather than v4.